packages/myst-cli/src/transforms/dois.ts

```diff
@@ -183,7 +183,7 @@ export async function getCitation(
   session: ISession,
   vfile: VFile,
   doiString: string,
-  node: GenericNode,
+  node?: GenericNode,
```

Comment:
Super minor change - everywhere `node` is used inside this function it is optional, so there's no reason it cannot be optional here.

This is helpful if we want to use `getCitation` in a context outside mdast.